queries/julia/highlights.scm

```diff
@@ -114,35 +114,236 @@
 
 ;; Builtins
 
+;; This list was generated with:
+;;
+;;  istype(x) = typeof(x) === DataType || typeof(x) === UnionAll
+;;  get_types(m) = filter(x -> istype(Base.eval(m, x)), names(m))
+;;  type_names = sort(union(get_types(Core), get_types(Base)))
+;;
 ((identifier) @type.builtin
  (#any-of? @type.builtin
-  "Type" "DataType" "Any" "Union" "UnionAll" "Tuple" "NTuple" "NamedTuple"
-  "Val" "Nothing" "Some" "Enum" "Expr" "Symbol" "Module" "Function" "ComposedFunction"
-  "Number" "Real" "AbstractFloat" "Integer" "Signed" "AbstractIrrational"
-  "Fix1" "Fix2" "Missing" "Cmd" "EnvDict" "VersionNumber" "ArgumentError"
-  "AssertionError" "BoundsError" "CompositeException" "DimensionMismatch"
-  "DivideError" "DomainError" "EOFError" "ErrorException" "InexactError"
-  "InterruptException" "KeyError" "LoadError" "MethodError" "OutOfMemoryError"
-  "ReadOnlyMemoryError" "OverflowError" "ProcessFailedException" "StackOverflowError"
-  "SystemError" "TypeError" "UndefKeywordError" "UndefRefError" "UndefVarError"
-  "StringIndexError" "InitError" "ExponentialBackOff" "Timer" "AsyncCondition"
-  "ParseError" "QuoteNode" "IteratorSize" "IteratorEltype" "AbstractRange"
-  "OrdinalRange" "AbstractUnitRange" "StepRange" "UnitRange" "LinRange" "AbstractDict"
-  "Dict" "IdDict" "WeakKeyDict" "ImmutableDict" "AbstractSet" "Set" "BitSet" "Pair"
-  "Pairs" "OneTo" " StepRangeLen" "RoundingMode" "Float16" "Float32" "Float64"
-  "BigFloat" "Bool" "Int" "Int8" "UInt8" "Int16" "UInt16" "Int32" "UInt32" "Int64"
-  "UInt64" "Int128" "UInt128" "BigInt" "Complex" "Rational" "Irrational" "AbstractChar"
-  "Char" "SubString" "Regex" "SubstitutionString" "RegexMatch" "AbstractArray"
-  "AbstractVector" "AbstractMatrix" "AbstractVecOrMat" "Array" "UndefInitializer"
-  "Vector" "Matrix" "VecOrMat" "DenseArray" "DenseVector" "DenseMatrix" "DenseVecOrMat"
-  "StridedArray" "StridedVector" "StridedMatrix" "StridedVecOrMat" "BitArray" "Dims"
-  "SubArray" "Task" "Condition" "Event" "Semaphore" "AbstractLniock" "ReentrantLock"
-  "Channel" "Atomic" "SpinLock" "RawFD" "IOStream" "IOBuffer" "AbstractDisplay" "MIME"
-  "TextDisplay" "PartialQuickSort" "Ordering" "ReverseOrdering" "By" "Lt" "Perm"
-  "Stateful" "CFunction" "Ptr" "Ref" "Cchar" "Cuchar" "Cshort" "Cstring" "Cushort"
-  "Cint" "Cuint" "Clong" "Culong" "Clonglong" "Culonglong" "Cintmax_t" "Cuintmax_t"
-  "Csize_t" "Cssize_t" "Cptrdiff_t" "Cwchar_t" "Cwstring" "Cfloat" "Cdouble" "Tmstruct"
-  "StackFrame" "StackTrace"))
+  "=>"
```

Comment:
The `=>` was returned as a type, but I'm not sure if we should leave it here. Only saw it after submitting the commit...

Reply:
We should remove it. That doesn't look like a type to me (or is it?)

Reply:
`=>` is an alias for `Pair`, which is already listed. It can be removed.